example/profiler/README.md

```diff
@@ -5,7 +5,9 @@ Please refer to [this link](http://mxnet.incubator.apache.org/faq/perf.html?high
 for visualizing profiling results and make sure that you have installed a version of MXNet compiled
 with `USE_PROFILER=1`.
 
-- profiler_executor.py. To run this example, simply type `python profiler_executor.py` in terminal.
+- profiler_executor.py. To run this example,
+1. clone mxnet-memonger (git clone https://github.com/dmlc/mxnet-memonger.git). Add path to mxnet-memonger to PYTHONPATH
```

Comment:
why do we need to clone the ``mxnet-memonger`` repo? 

And in this line - "Add path to mxnet-memonger to PYTHONPATH" do you mean, "Add ``mxnet-memonger`` folder to PYTHONPATH"

Reply:
As of now, this is the way to use mxnet-memonger (https://github.com/dmlc/mxnet-memonger/issues/4). Alternative is to add it to 3rdparty folder maybe?

Yes, I meant 'export PYTHONPATH=$PYTHONPATH:/path/to/mxnet-memonger'. Something like this.

Reply:
then please change it accordingly.